### PR TITLE
chore(ci): make sure to create a GitHub release beforehand

### PR DESCRIPTION
**Pull request by barmac**
Related to the duplicated release for Camunda Modeler v4.0.0-alpha.3
This prevents racing condition.

### CODE REVIEW

**Comment by barmac** [comment]
I verified that it creates a release on my private repo.

**Comment by barmac** [comment]
For the used tool, see: https://hub.github.com/hub-release.1.html

File: .travis.yml [hunk 1]
```diff
@@ -80,6 +86,8 @@ jobs:
 
 stages:
   - name: test
+  - name: "pre distro"
+    if tag =~ ^v\d
```
Comment (oguzeroglu) [inline .travis.yml:90]: Not exactly sure but do we maybe need `:` here after `if`, we have it like that in line 92:

`if: tag =~ ^v\d`

Wanna make sure if this is not a typo.

Comment (nikku) [reply .travis.yml:90]: Good catch.

Comment (barmac) [reply .travis.yml:90]: Indeed.

Comment (barmac) [reply .travis.yml]: Fixed.

Comment (oguzeroglu) [reply .travis.yml]: ⭐

**Comment by nikku** [comment]
Good candidate for merging into the Zeebe Modeler.

**Comment by barmac** [comment]
Why not, let's port it.

**Pull request merged**